android/app/build.gradle

```diff
@@ -374,7 +374,7 @@ dependencies {
         // androidx.appcompat:appcompat:1.4.0+ breaks TextInput. This was fixed
         // in react-native 0.68. For more details, see
         // https://github.com/facebook/react-native/issues/31572.
-        implementation "androidx.appcompat:appcompat:1.3.1"
+        implementation(["androidx.appcompat", "appcompat", "1.3.1"].join(":"))
```

Comment:
isn't there a way to touch some Renovate config instead of this workaround?

Reply:
We still want it to upgrade the other instance (in [`dependencies.gradle`](https://github.com/microsoft/react-native-test-app/blob/bf952e4e4ae0fe9662beb2a3dd88cfd064d9a560/android/dependencies.gradle#L36)). Not aware of any configuration that would allow pinning one instance and not the other.

Reply:
ok